addons/quickmount/initSettings.sqf

```diff
@@ -0,0 +1,36 @@
+[
+    QGVAR(enabled),
+    "CHECKBOX",
+    localize ELSTRING(common,Enabled),
+    format ["ACE %1", localize LSTRING(Category)],
```

Comment:
`LLSTRING` and `LELSTRING` could be used here and anywhere

Reply:
Could simply delete the `localize` , because the settings system can translate the strings on its own.

Reply:
btw just checked, subcategory is not automatically localized

Reply:
Issue report then.

Reply:
fixed in https://github.com/CBATeam/CBA_A3/pull/1003